Exercise5/toBinary.js

```diff
@@ -0,0 +1,15 @@
+// Function that converts a number to its binary representation
+
+const toBinary = (input) => {
+  // Check if the input is a number
+  if (typeof input !== "number")
+    throw new Error(
+      `Your input is of type ${typeof input}. Please input a number.`
+    );
+  return input.toString(2);
```

Comment:
The returned value when the parameter value is a negative number is not realistic.
eg: toBinary(-9) // -1001
**N.B**:  In binary number sign are represented as follow: 
- 1 is added in front if it is a negative number
- 0 is added in front if it is a positive number